go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/clarketm/json v1.14.1
 	github.com/coreos/ignition/v2 v2.10.1
 	github.com/docker/distribution v2.7.1+incompatible
+	github.com/eranco74/cluster-api-provider-agent/api v0.0.0-20211202085429-b633556695e5
```

Comment:
can we now update to use https://github.com/openshift/cluster-api-provider-agent

Reply:
Updated